### DISCUSSION TITLE
REMOVE pilot quals. delay

### DISCUSSION THREAD

**Discussion by djnigelcage** [Xeno69/Domination#541] [Q&A]
hi

first of all t Xeno, awesome PBO, sir

hats down to you

i was able to locate and set defaults for 99% of stuff i wanted to change

the one last final thing escaping me is the delay for how soon a player can qualify to fly

can someone please point me to the right file or script where this function is stored?

thank you

BravoKilo9

**Comment by Xeno69** [comment]
Check
https://github.com/Xeno69/Domination/wiki/Params#d_autokicktime--air-vecs-autokick-time

**Reply by djnigelcage** [reply]
i have seen the parameters and i have the ONLY PILOTS disabled and the HELIS AND ARMOR disabled.... but when the game starts and i drive over to aircraft, i get YOU WILL BE QUALIFIED TO FLY IN 30 MINUTES message

i dont seem to understand which file or parameter controls that delay

**Reply by Xeno69** [reply]
I think there is an issue which I have to fix fist, the auto kick time is completely ignored when some kind of database is used (SQL or profileNameSpace)

**Comment by djnigelcage** [comment]
I think i only have that problem cause i had TK penalties enabled and i was
blowing up things to test new respawn settings

Cause since i disabled all the restrictions i dont have that delay issue
anymore

I mainly play alone so i edited to my preferences

Removed all restrictions and added some more ai recruiting from mhq so i
can have some guys with me in the combat zone without having to haul them
from base

For some reason i dont get the artillery stuff

Do only those two arrillery guys have the ability to call in strikes?

**Reply by Xeno69** [reply]
If you have enabled AI recruitment (with ai) the artillery is available for all players.
But to be able to mark an artillery target you need laser designator (after marking a target you still need to call on arty)

If AI is enabled all players are also medics and engineers.

**Comment by djnigelcage** [comment]
I got everything how i like it as far as the present settings

What i would love though for my lonely assaults is the ai recruit in both
mhq vehicles

I adden bon_recruit_units script to them and it works, but when they get
blown up and respawn, the new vehicle no longer has the init code in it

How does you ai recruiting work and can it be assigned to the mhq vehicles
and persist even after those vwhicles respawning?

Many thanks for the replies
